go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/solarwinds/grpchan
 
-go 1.14
+go 1.15
```

Comment:
actually, it's strange that here is 1.15 and working with lib with 1.13. I don't understand go modules :(